src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -1426,7 +1429,7 @@ def destroy_containers(self, containers):
 
         """
         error_list = []
-        if containers:
+        if containers and not self.skip_destroy_containers:
```

Comment:
I'd recommend not adding a new `TestWithServers` attribute when this can be handled with `TestContainer` attributes.
```suggestion
        if containers:
```